testing/xfpga/test_sysfs_c.cpp

```diff
@@ -179,6 +179,7 @@ TEST_P(sysfsinit_c_p, sysfs_initialize) {
         devs.erase(id);
       }
     }
+    return FPGA_OK;
   };
```

Comment:
extraneous ;

Reply:
That's actually the end of assignment of the the lambda function to `cb` variable on line 172.

Reply:
ah, my bad.